src/components/SelectionPage/CarouselContainer.jsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+import { FaCircleChevronLeft, FaCircleChevronRight } from "react-icons/fa6";
+
+import useUrlStore from "../../store/store";
+
+function CarouselContainer() {
+  const { videoUrls } = useUrlStore();
+  const urlVideoList = Object.values(videoUrls);
+
+  const [currentIndex, setCurrentIndex] = useState(0);
```

Comment:
훅은 `function` 최상단으로 모두 올리는 건 어떨까요?
```suggestion
function CarouselContainer() {
  const { videoUrls } = useUrlStore();
  const [currentIndex, setCurrentIndex] = useState(0);

  const urlVideoList = Object.values(videoUrls);
```